### PR TITLE
[TRAVIS] Don't follow log from libdrizze-redux container

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Fixes and issue where builds failed on travis when the
container log was written to the build log

### CODE REVIEW

File: travis_configure.sh [hunk 1]
```diff
@@ -117,8 +117,6 @@ run_tests()
 {
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
```
Comment (daniel-zullo) [inline travis_configure.sh:118]: what about "Don't follow `the` log `output` from libdrizze-redux container" as a commit title (if it isn't too long)

Also there is a typo in `and` in the commit message. s/and/an

**Review by daniel-zullo — COMMENTED** [review]
A cosmetic comment and LGTM

**Comment by andreas-bok-sociomantic** [comment]
Updated ci title

**Comment by daniel-zullo** [comment]
> Also there is a typo in and in the commit message. s/and/an

you didn't fix the typo though

**Comment by andreas-bok-sociomantic** [comment]
> you didn't fix the typo though

Sorry, fixed now

**Comment by andreas-bok-sociomantic** [comment]
The checks fails because I cancelled the build. (No code code changes)

**Review by daniel-zullo — APPROVED** [review]

**Pull request merged**